website/server/controllers/api-v3/auth.js

```diff
@@ -239,17 +241,39 @@ function _passportFbProfile (accessToken) {
   });
 }
 
+function _passportGoogleProfile (accessToken) {
+  return new Bluebird((resolve, reject) => {
```

Comment:
Since it's just a normal promise here (nothing Bluebird specific), just use `Promise`